<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalariaAutoDetection</class>
 <widget class="qMRMLWidget" name="MalariaAutoDetection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>580</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>ðŸ“¥ EntrÃ©es</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image de microscopie:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ModÃ¨le de dÃ©tection:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="modelSelection">
        <property name="toolTip">
         <string>Select the detection model</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="analyzeButton">
        <property name="toolTip">
         <string>Start automatic detection</string>
        </property>
        <property name="styleSheet">
         <string>QPushButton {
           background-color: #BDBDBD;
           color: #757575;
           font-weight: bold;
           padding: 14px;
           border-radius: 5px;
           font-size: 14px;
           border: none;
         }
         QPushButton:enabled {
           background-color: #4CAF50;
           color: white;
         }
         QPushButton:enabled:hover {
           background-color: #45A049;
         }
         QPushButton:enabled:pressed {
           background-color: #388E3C;
         }</string>
        </property>
        <property name="text">
         <string>â–¶ Commencer l'analyse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputImageNodeSelector">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLVolumeNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="editEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>ðŸ“Š RÃ©sultats</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_outputs">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QGroupBox" name="analyticsGroupBox">
        <property name="styleSheet">
         <string>QGroupBox {
           font-weight: bold;
           font-size: 13px;
           border: 2px solid #E0E0E0;
           border-radius: 5px;
           margin-top: 12px;
           padding-top: 12px;
           color: #333333;
         }
         QGroupBox::title {
           subcontrol-origin: margin;
           left: 12px;
           padding: 0 8px;
           background-color: white;
         }</string>
        </property>
        <property name="title">
         <string>DÃ©tections</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_analytics">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_parasites">
           <item>
            <widget class="QLabel" name="label_parasiteCount">
             <property name="styleSheet">
              <string>QLabel { 
                font-size: 12px; 
                font-weight: normal;
                color: #555555;
              }</string>
             </property>
             <property name="text">
              <string>ðŸ¦  Parasites</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_parasites">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="parasiteCountLabel">
             <property name="styleSheet">
              <string>QLabel { 
                font-weight: bold; 
                font-size: 18px; 
                color: #D32F2F;
                padding: 6px 12px;
                background-color: #FFEBEE;
                border-radius: 4px;
                min-width: 60px;
              }</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_leukocytes">
           <item>
            <widget class="QLabel" name="label_leukocyteCount">
             <property name="styleSheet">
              <string>QLabel { 
                font-size: 12px; 
                font-weight: normal;
                color: #555555;
              }</string>
             </property>
             <property name="text">
              <string>âšª Leucocytes</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_leukocytes">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="leukocyteCountLabel">
             <property name="styleSheet">
              <string>QLabel { 
                font-weight: bold; 
                font-size: 18px; 
                color: #1976D2;
                padding: 6px 12px;
                background-color: #E3F2FD;
                border-radius: 4px;
                min-width: 60px;
              }</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_density">
           <item>
            <widget class="QLabel" name="label_parasiticDensity">
             <property name="styleSheet">
              <string>QLabel { 
                font-size: 12px; 
                font-weight: normal;
                color: #555555;
              }</string>
             </property>
             <property name="text">
              <string>ðŸ“ˆ DensitÃ© (DP)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_density">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="parasiticDensityLabel">
             <property name="styleSheet">
              <string>QLabel { 
                font-weight: bold; 
                font-size: 18px; 
                color: #388E3C;
                padding: 6px 12px;
                background-color: #E8F5E9;
                border-radius: 4px;
                min-width: 60px;
              }</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Exporter les rÃ©sultats au format CSV</string>
        </property>
        <property name="styleSheet">
         <string>QPushButton {
           background-color: #BDBDBD;
           color: #757575;
           font-weight: bold;
           padding: 12px;
           border-radius: 4px;
           font-size: 13px;
           border: none;
         }
         QPushButton:enabled {
           background-color: #2196F3;
           color: white;
         }
         QPushButton:enabled:hover {
           background-color: #1976D2;
         }
         QPushButton:enabled:pressed {
           background-color: #1565C0;
         }</string>
        </property>
        <property name="text">
         <string>ðŸ’¾ Exporter (CSV)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="advancedCollapsibleButton">
     <property name="text">
      <string>âš¡ AvancÃ©</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_advanced">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_advancedPlaceholder">
        <property name="styleSheet">
         <string>QLabel { 
           color: #9E9E9E; 
           font-style: italic;
           padding: 20px;
           font-size: 11px;
         }</string>
        </property>
        <property name="text">
         <string>ParamÃ¨tres avancÃ©s disponibles prochainement</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>MalariaAutoDetection</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputImageNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
